lib/json2csv.js

```diff
@@ -280,33 +287,57 @@ function createColumnContent(params, str) {
 }
 
 /**
- * Performs the unwind logic if necessary to convert single JSON document into multiple rows
- * @param params
+ * Performs the unwind recursively in specified sequence
+ * 
+ * @param {Array} originalData The params.data value. Original array of JSON objects
+ * @param {String[]} unwindPaths The params.unwindPath value. Unwind strings to be used to deconstruct array
+ * @returns {Array} Array of objects containing all rows after unwind of chosen paths
  */
-function createDataRows(params) {
-  var dataRows = params.data;
-
-  if (params.unwindPath) {
+function createDataRows(originalData, unwindPaths) {
+  var dataRows = originalData;
+  if (unwindPaths.length) {
     dataRows = [];
```

Comment:
Looks like dataRows gets overwritten right away, why not set it to an empty array on line 297, and just return originalData if `unwindPaths.length` is falsy.